docs/versioned_docs/version-2.3.0/api/miscellaneous/getRelativeCoords.md

```diff
@@ -0,0 +1,49 @@
+---
+id: getRelativeCoords
+title: getRelativeCoords
+sidebar_label: getRelativeCoords
+---
+
+Determines the location on the screen, relative to the given view. It might be useful, when there are only absolute coordinates available and you need coordinates relative to the parent.
```

Comment:
```suggestion
Determines the location on the screen, relative to the given view. It might be useful when there are only absolute coordinates available and you need coordinates relative to the parent.
```